src/engine/ruleset/environments/wazuh-environment.yml

```diff
@@ -61,5 +61,8 @@ decoders:
   # Service decoders
   - decoder/syslog/0
 
+  # Sophos decoders
+  - decoder/sophos-antispam/0
```

Comment:
Improve-desing: Please respect the way in which decoders have been located and place them in alphabetical order